tests/smt/smt_server_install.pm

```diff
@@ -42,9 +42,11 @@ sub run {
     wait_still_screen(1);
     type_string get_var('SMT_PASSWORD');
     wait_still_screen(1);
-    wait_screen_change { send_key "alt-t" };
-    assert_screen "smt-test-succ", 120;
-    wait_screen_change { send_key "ret" };
+    do {
+        send_key "alt-t";
+        check_screen "smt-test-succ", 120;
+        send_key "ret";
+    } until match_has_tag 'smt-test-succ';
```

Comment:
One comment here, it may cause  unlimited loop if no needle match?, correct?

Reply:
Agree with Richard, maybe having a fixed max number of retry is a good idea? To avoid this looping until max job time

Reply:
done